src/appleseed/renderer/kernel/rendering/progressive/progressiveframerenderer.cpp

```diff
@@ -360,6 +360,26 @@ namespace
             print_sample_generators_stats();
         }
 
+        void print_settings() const override
+        {
+            RENDERER_LOG_INFO(
+                        "generic renderer settings:\n"
+                        "  spectrum mode                 %s\n"
+                        "  thread count                  %zu\n"
+			            "  max samples                   " FMT_UINT64 "\n"
```

Comment:
Fix indentation (tabs aren't allowed, always use 4 spaces instead).

Reply:
Oh ok, as in python.

Reply:
It would be nice to use something similar to [editorconfig](http://editorconfig.org/) since each project have their own requirements.

Reply:
Yes, absolutely! I've been thinking about providing an EditorConfig file for a while but never had time to do it. I'll create a ticket, feel free to do it if you're interested :)